packages/@aws-cdk/aws-route53/README.md

```diff
@@ -123,4 +123,51 @@ you know the ID and the retrieval for the `zoneName` is undesirable.
 const zone = HostedZone.fromHostedZoneId(this, 'MyZone', {
   hostedZoneId: 'ZOJJZC49E0EPZ',
 });
+```
+
+
+## VPC Endpoint Service Private DNS
+
+When you create a VPC endpoint service, AWS generates endpoint-specific DNS hostnames that consumers use to communicate with the service.
```

Comment:
Oh I love this section. That's because I badly need it, I have no idea why you would care about this otherwise :)